evm-node/Dockerfile

```
@@ -0,0 +1,10 @@
+# Ubuntu is fine even though it's not the smallest image
+FROM ghcr.io/foundry-rs/foundry:latest
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)